From 746662562995125ef7fb2c294300b0bd061b1251 Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Tue, 10 Dec 2024 16:39:31 +0000
Subject: [PATCH] Revert "PCI: Warn if no host bridge NUMA node info"

This warning doesn't mean anyting on our platform and
the warning causes confusion.

See: https://forums.raspberrypi.com/viewtopic.php?p=2276125#p2276125

This reverts commit ad5086108b9f0361929aa9a79cf959ab5681d249.

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 drivers/pci/probe.c | 3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -967,9 +967,6 @@ static int pci_register_host_bridge(stru
 	else
 		pr_info("PCI host bridge to bus %s\n", name);
 
-	if (nr_node_ids > 1 && pcibus_to_node(bus) == NUMA_NO_NODE)
-		dev_warn(&bus->dev, "Unknown NUMA node; performance will be reduced\n");
-
 	/* Coalesce contiguous windows */
 	resource_list_for_each_entry_safe(window, n, &resources) {
 		if (list_is_last(&window->node, &resources))
